src/scss/components/_privacy-checkbox.scss

```diff
@@ -5,13 +5,17 @@
 
   &__label {
     @extend .#{prefix(checkbox)}__label;
+  }
 
+  &__wyswyg-text {
     // This is needed because the label is populated with the value coming from a WYSIWYG editor
-    // which adds <p></p> around the label text. This tag gets a margin which looks off.
+    // which adds <p></p> around each paragraph in the text. This tag gets a margin which looks off.
     // So here it is overwritten.
     p {
-      margin: 0;
+      margin: 0 0 1em 0;
     }
+
+    p:last-child { margin-bottom: 0 !important; }
```

Comment:
You can simplify this using the [`body` mixin](https://maykinmedia.github.io/microscope-sass/#typography-mixin-body) mixin

```scss
p {
    @include body('auto');  // sets margin-bottom automatically except for the last child
}
```

The advantage is that it doesn't use any magic numbers (`1em`) and instead uses the globally configured spacing configuration. If that is too much (I think it's 20px?) you can use the lower level `@include margin('auto', $value-mobile: 1em);` instead.

Reply:
I think it looks okay with the margin given by `@include margin('auto')`, so i didn't use the smaller value! :slightly_smiling_face: 

![Screenshot from 2022-02-03 16-57-48](https://user-images.githubusercontent.com/19154114/152379208-81c6843b-e652-438a-b592-d819edd8fdda.png)